### PR TITLE
Send email on validation

### PR DESCRIPTION
**Pull request by Rhian**
### Description of change

This sends a notification via Notify to the applicant's email once their application has been validated and makes the "email not sent" error message for Determination more generic

### Story Link

https://trello.com/c/VK9OMXtT/32-notify-applicant-when-application-is-marked-as-valid

### CODE REVIEW

File: app/controllers/planning_applications_controller.rb [hunk 1]
```diff
@@ -127,8 +128,6 @@ def validate_documents
     end
   end
 
-private
```
Comment (james) [inline app/controllers/planning_applications_controller.rb:130]: Why did we un-`private` these methods?

File: spec/system/planning_applications/reviewing_spec.rb [hunk 2]
```diff
@@ -297,7 +297,7 @@
           click_link "Publish the recommendation"
           click_button "Determine application"
 
-          expect(page).to have_content("The Decision Notice cannot be sent. Please try again later.")
+          expect(page).to have_content("The email cannot be sent. Please try again later.")
```
Comment (james) [inline spec/system/planning_applications/reviewing_spec.rb:300]: Should we also have a feature spec that checks we send the email when validating?

Comment (Rhian) [reply spec/system/planning_applications/reviewing_spec.rb:300]: I've added a line into the validation spec to verify that the delivered mails count increments by 1 on validation, but if you can think of a more complete way to do it, please let me know. I figured we didn't need to validate the content as we are already doing this in the mailer spec.

File: app/controllers/planning_applications_controller.rb (hunk 1 above)
Comment (Rhian) [reply app/controllers/planning_applications_controller.rb:130]: ..entirely by accident 😩 Thank you very much for picking up on this!

**Review by james — APPROVED** [review]
Looks good to me once you've fixed rubocop :-)

**Pull request merged**